src/index.js

```diff
@@ -6,6 +6,8 @@ const RIGHT = '-->';
 const ENCODE = [
   ['&', '&amp;'],
   ['>', '&gt;'],
+  ['<', '&lt;'],
```

Comment:
note that I’m pretty sure this isnt needed, it just increases the HTML size for no benefit.